site/content/_posts/2024-03-10-march_2024_meet.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "General Meeting on March 28, 2024"
+date: 2024-03-11T01:53:05.442Z
+categories: community
+---
+We will be having a Community Meeting in March!
```

Comment:
I love the term Community Meeting, and it would be great to adopt.   That said, it seems like we have been calling it a "General Meeting" pretty consistently everywhere else.  

```suggestion
We will be having a General Meeting in March!
```